hub/constants.py

```diff
@@ -34,8 +35,14 @@
 DEFAULT_LOCAL_CACHE_SIZE = 0
 
 
+# meta is 100% required hub-defined meta
```

Comment:
`meta is hub-defined information, necessary for hub Datasets to function`. Let's avoid recursive explanations 😄

Reply:
looool